### DISCUSSION TITLE
No envía los botones

### DISCUSSION THREAD

**Discussion by Coaxus-ux** [codigoencasa/builderbot#781] [Q&A] [answered]
Estoy usando baileys y al momento de querer enviar un botón no se envía nada, ni siquiera el mensaje de cabecera. He probado tanto en mi celular "Ios" como en el navegador. 
`
const ButtonsTest = addKeyword(["botones"]).addAnswer(
  "🙌 Hola bienvenidos",
  {
    buttons: [
      {
        body: "📄 Documentación",
      },
      {
        body: "📚 Tutorial",
      },
      {
        body: "🤪 Discord",
      },
    ],
  }
);
`

**Comment by leifermendez** [comment, marked as answer]
Buenas el tema se botones con los proveedores gratuitos luego de la update de WhatsApp dejaron de funcionar te recomiendo usar Meta para los botones ya que es una solución estable

**Reply by CristhianMunozMunoz** [reply]
hola Leifer espero te encuentres bien, quisiera saber si se pueden integrar mas de 3 botones, estoy usando meta como proveedor pero al momento de agregar un cuarto boton no me aparece, no se si me puedas dar una mano, gracias

**Comment by sferreradediego** [comment]
Observo que si usamos Meta, tiene un coste en caso de desear como número remitente el nuestro, ¿existe alguna alternativa manteniendo el uso de botones?. Gracias.
